static/scss/custom.scss

```diff
@@ -1,11 +1,5 @@
-@import url('https://fonts.googleapis.com/css?family=Montserrat&display=swap');
-@import "../../node_modules/bootstrap-scss/bootstrap";
 @import "_variables";
```

Comment:
o ile dobrze pamietam konwencje to ten podkreslnik jest w nazwie pliku, ale w imporcie jest niepotrzebny
to jest tylko zaznaczenie, ze plik nie jest samodzielny a przeznaczony do importu